firstvoices/backend/models/constants.py

```diff
@@ -29,3 +29,11 @@ class AppRole(models.IntegerChoices):
     # enum intentionally has gaps to allow future changes to keep sequential order
     STAFF = 0, _("Staff")
     SUPERADMIN = 10, _("Superadmin")
+
+
+class JoinRequestStatus(models.IntegerChoices):
+    # enum intentionally has gaps to allow future changes to keep sequential order
+    PENDING = 0, _("Pending")
+    APPROVED = 10, _("Approved")
+    REJECTED = 20, _("Rejected")
+    CANCELLED = 30, _("Cancelled")
```

Comment:
This list of statuses looks great, but I am pondering whether there are any opportunities to make the ordering more useful. It says "integer" so we should be able to do negatives... maybe we could something like

CANCELLED = -20
REJECTED = -10
PENDING = 0
APPROVED = 10

I can think of scenarios for wanting >=REJECTED, and for >=PENDING, so maybe it's useful to sort them?

Reply:
And a minor thing: this is so specific to join requests that I think it could go in join_request.py